### PR TITLE
Add collapsible component

### PR DESCRIPTION
**Pull request by Poltergeist**
With this component it is possible to create a
section which is collapsible and is hidden behind
a label this label can be configured with the
label prop. The label prop expects a string

Example:

```
<CollapsibleContainer label="Advanced Setting">
  ... Some Content
</CollapsibleContainer>
```

Closed:
image

Open:
image

### CODE REVIEW

File: src/js/components/CollapsibleContainer.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  }
+
+}
+
+CollapsibleContainer.defaultProps = {
+  label: ''
+};
+
+CollapsibleContainer.propTypes = {
+  label: React.PropTypes.string
```
Comment (nLight) [inline src/js/components/CollapsibleContainer.js]: Should we require a label?

Comment (orlandohohmeier) [inline src/js/components/CollapsibleContainer.js]: Let's make this a required prop!

**Comment by Poltergeist** [comment]
@orlandohohmeier @nLight Comment haz been addressed.

**Comment by orlandohohmeier** [comment]
LGTM! 🚢

**Pull request merged**